olm/prepare-community-operators-update.sh

```diff
@@ -25,23 +25,28 @@ do
   cd "${packageBaseFolderPath}"
 
   packageFolderPath="${packageBaseFolderPath}/deploy/olm-catalog/${packageName}"
+  sourcePackageFilePath="${packageFolderPath}/${packageName}.package.yaml"
   communityOperatorsLocalGitFolder="${packageBaseFolderPath}/generated/community-operators"
+  lastPackagePreReleaseVersion=$(yq -r '.channels[] | select(.name == "stable") | .currentCSV' "${sourcePackageFilePath}" | sed -e "s/${packageName}.v//")
```

Comment:
Script should check if `yq` is installed (on line 13?) and fail before doing anything if `yq` is not installed with a message telling the user where to get it (as there are many yq flavours out there).

Reply:
Fixed in commit https://github.com/eclipse/che-operator/pull/60/commits/474197cbae315ac482e7b5890986727583d2c20c